### PR TITLE
Extend appinfo configuration

### PR DESCRIPTION
**Pull request by drahosistvan**
On WebOS, you are able to configure application icon background, splash screen, and hover image from appinfo.json.
With this pull request, developers are able to configure the webOS builds better.

This pull request is connected to my [cordova-tv-webos](https://github.com/Samsung/cordova-tv-webos/pull/1) request.

### CODE REVIEW

File: tasks/packager/tv-webos.js [hunk 1]
```diff
@@ -101,11 +101,17 @@ function validateWebosVersion(version) {
...
-    console.log('      > vendor      : ' + userData.vendor);
-    console.log('      > resolution  : ' + userData.resolution);
+    console.log('      > id                 : ' + userData.id);
+    console.log('      > name               : ' + userData.name);
+    console.log('      > version            : ' + userData.version);
+    console.log('      > vendor             : ' + userData.vendor);
+    console.log('      > icon               : ' + userData.icon);
+    console.log('      > largeIcon          : ' + userData.largeicon);
+    console.log('      > resolution         : ' + userData.resolution);
+    console.log('      > bgColor            : ' + userData.bgcolor);
```
Comment (prozanne) [inline tasks/packager/tv-webos.js]: We decided to show stored information only for mandatory items we think.
If all kind of attributes are shown, log might be complex.

So, can you rollback this to have only id, name, version, vendor, resolution as previous?

File: tasks/packager/tv-webos.js [hunk 2]
```diff
@@ -186,6 +192,38 @@ function askUserData(cordovaConf, callback, versionOnly, userData) {
...
+            message: 'Splash screen background path (default is \'img/logo.png\')',
+            default: 'img/logo.png',
+            validate: function(input) {
+                return typeof(input) === 'string' ? true : 'invalid path';
+            }
+        }, {
+            type: 'input',
+            name: 'bgcolor',
+            message: 'Background color for app',
+            default: 'red',
```
Comment (prozanne) [inline tasks/packager/tv-webos.js]: Could you change default color to white or black (achromatic color)?
I think it's better to have an inconspicuous color.

File: tasks/packager/tv-webos.js [hunk 3]
```diff
@@ -186,6 +192,38 @@ function askUserData(cordovaConf, callback, versionOnly, userData) {
...
+            validate: function(input) {
+                return typeof(input) === 'string' ? true : 'invalid color';
+            }
+        }, {
+            type: 'input',
+            name: 'iconcolor',
+            message: 'Icon color for app',
+            default: 'red',
```
Comment (prozanne) [inline tasks/packager/tv-webos.js]: Could you change default color to white or black (achromatic color)?
I think it's better to have an inconspicuous color.

**Review by prozanne — CHANGES_REQUESTED** [review]
Would you consider these reviews?

File: tasks/packager/tv-webos.js (hunk 1 above)
Comment (drahosistvan) [reply tasks/packager/tv-webos.js]: I modified the console log. Now, it is untouched.

File: tasks/packager/tv-webos.js (hunk 2 above)
Comment (drahosistvan) [reply tasks/packager/tv-webos.js]: Default color is modified to white.

File: tasks/packager/tv-webos.js (hunk 3 above)
Comment (drahosistvan) [reply tasks/packager/tv-webos.js]: Default color is modified to white.

**Review by prozanne — APPROVED** [review]
Thank you for your contribution!!!

**Pull request merged**